### ISSUE TITLE
Typo in README file

### ISSUE THREAD

**Issue by StaelTchinda** [DLR-RM/curvature#6]
It seems to be a typo in the code snippet of the [Get Started section](https://github.com/DLR-RM/curvature/blob/main/README.rst#get-started). The loss is computed based on the `logits` and the `sampled_labels` (which are obtained from the `logits`) however the (groundtruth) `labels` should be used instead of the `sampled_labels`.

```
...
for images, labels in tqdm.tqdm(train_data):
    logits = model(images.to(device))

    # We compute the 'true' Fisher information matrix (FIM),
    # by taking the expectation over the model distribution.
    # To obtain the empirical FIM, just use the labels from
    # the data distribution directly.
    dist = torch.distributions.Categorical(logits=logits)
    sampled_labels = dist.sample()

    loss = criterion(logits, sampled_labels)
...
```

**Comment by jlee4176901** [comment]
- there is difference between the concept of empirical fisher information matrix, which is known to be biased.
- possible reference to read: https://proceedings.neurips.cc/paper/2019/file/46a558d97954d0692411c861cf78ef79-Paper.pdf
- so, it is not a typo. So, I will close the issue.
